src/service/url-replacements-impl.js

```diff
@@ -116,38 +128,41 @@ export class GlobalVariableSource extends VariableSource {
     /** @const {!./viewport/viewport-impl.Viewport} */
     const viewport = Services.viewportForDoc(this.ampdoc);
 
+    const set = this.set.bind(this);
+    const setAsync = this.setAsync.bind(this);
+
     // Returns a random value for cache busters.
-    this.set('RANDOM', () => Math.random());
```

Comment:
Ditto bad codesize wins. This _increased_ the brotli size by 70 bytes.

`this.` is **so frequent** in JS it will have an optimized representation in the compressor.

Reply:
will undo. was going by gzip size.